lib/folio_sync/archives_space/manual_updater.rb

```diff
@@ -0,0 +1,89 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+# This class is meant to be used in a one-time manual step
+module FolioSync
+  module ArchivesSpace
+    class ManualUpdater
+      PAGE_SIZE = 200
+
+      def initialize(instance_key)
+        @logger = Logger.new($stdout)
+        @aspace_client = FolioSync::ArchivesSpace::Client.new(instance_key)
+        @folio_client = FolioSync::Folio::Client.instance
+        @instance_key = instance_key
+        @csv_file_path = "tmp/#{instance_key}_updated_aspace_resources_#{Time.zone.now.strftime('%Y%m%d%H%M%S')}.csv"
```

Comment:
```suggestion
        @csv_file_path = Rails.root.join("tmp/#{instance_key}_updated_aspace_resources_#{Time.zone.now.strftime('%Y%m%d%H%M%S')}.csv")
```
In the context of a Rails app, I like to use the Rails root directory's `tmp` subdirectory for exports like this rather than supplying a relative path to the working directory or the OS tmp directory.  It's slightly more reliable across systems (even though we probably won't ever have to worry about using this app on a Windows machine, and in most cases would be running code with the git repo root as the working directory).